tokio/src/runtime/metrics/io.rs

```diff
@@ -0,0 +1,21 @@
+use std::sync::atomic::{AtomicU64, Ordering::Relaxed};
+
+#[derive(Default)]
+pub(crate) struct IoDriverMetrics {
+    pub(super) fd_count: AtomicU64,
+    pub(super) ready_count: AtomicU64,
+}
+
+impl IoDriverMetrics {
+    pub(crate) fn incr_fd_count(&self) {
+        self.fd_count.fetch_add(1, Relaxed);
```

Comment:
Since there is only one "producer" of metrics instead of `fetch_add`, i would do a load / store combo. It is cheaper.